command/command.go

```diff
@@ -84,6 +85,11 @@ var Commands = map[string]Command{
 		Exec:  poll,
 		Usage: pollUsage,
 	},
+	"blocklist": modPrivateOnly(moderatorOnly(Command{
```

Comment:
we probably want this in reverse order, this will prompt a non-moderator to run  the command in a moderator-internal channel

```suggestion
	"blocklist": moderatorOnly(modPrivateOnly(Command{
```